packages/aws-cdk/test/api/logs/logs-monitor.test.ts

```diff
@@ -0,0 +1,103 @@
+import { blue, yellow } from 'colors/safe';
+import { CloudWatchLogEventMonitor } from '../../../lib/api/logs/logs-monitor';
+import { sleep } from '../../integ/helpers/aws';
+import { MockSdk } from '../../util/mock-sdk';
+
+let sdk: MockSdk;
+let stderrMock: jest.SpyInstance;
+let monitor: CloudWatchLogEventMonitor;
+beforeEach(() => {
+  jest.useFakeTimers('legacy');
+  monitor = new CloudWatchLogEventMonitor(new Date(T100));
+  monitor.activate();
+  stderrMock = jest.spyOn(process.stderr, 'write').mockImplementation(() => { return true; });
+  sdk = new MockSdk();
+});
+
+afterAll(() => {
+  jest.useRealTimers();
+  stderrMock.mockRestore();
```

Comment:
Should we call ` monitor.deactivate();` here? So that we stop the `setTimeout()` calls?